ICT4DNews/app/src/main/java/at/ict4d/ict4dnews/server/ApiJsonSelfHostedWPService.kt

```diff
@@ -5,15 +5,20 @@ import at.ict4d.ict4dnews.models.wordpress.WordpressAuthor
 import at.ict4d.ict4dnews.models.wordpress.WordpressMedia
 import io.reactivex.Flowable
 import io.reactivex.Single
+import org.threeten.bp.LocalDateTime
+import org.threeten.bp.format.DateTimeFormatter
 import retrofit2.Call
 import retrofit2.http.GET
 import retrofit2.http.Path
 import retrofit2.http.Query
 
 interface ApiJsonSelfHostedWPService {
 
-    @GET("http://www.ict4d.at/wp-json/wp/v2/posts?per_page=10")
```

Comment:
i think if we add `http://www.ict4d.at/` as a base url in our `Retrofit` provider methods and only place `wp-json/wp/v2/posts` necessary link to our api methods it will work and look more beautiful ;)

Reply:
Hi, valid suggestion! We will handle this case later on, since we will have many base URLs in the future. Each per news source.  I will merge the changes into develop...